libmscore/mscore.h

```diff
@@ -70,7 +70,12 @@ static const int MAX_TAGS = 32;
 static constexpr qreal INCH      = 25.4;
 static constexpr qreal PPI       = 72.0;           // printer points per inch
 static constexpr qreal SPATIUM20 = 5.0;
-static constexpr qreal DPI       = 72.0;
+#ifdef Q_OS_WIN
+static constexpr qreal DPIFACTOR = 10.0;
```

Comment:
in master at this spot we see DPI_F = 5 and for all platforms.
It is only used here and in sym.cpp, not in those other files you use DPI_FACTOR in.

Reply:
Yes, but in master also SPATIUM20 is modified by DPI_F.
See https://github.com/musescore/MuseScore/commit/4ac407d90ec1d5663f0c753acc3ef5f4bcca2422
I didn't want to change the code too much for fear of unexpected regressions.
If I change only mscore.h by adding a DPIFACTOR also to SPATIUM20, all score objects get reduced by a factor DPIFACTOR.

By the way, I noticed that the exported svg seems right, but the page is scaled up by a factor DPIFACTOR (the page size of the exported pdf on the other hand looks correct).